alica_engine/include/engine/constraintmodul/VariableSyncModule.h

```diff
@@ -45,6 +45,8 @@ class VariableSyncModule
     VariableSyncModule& operator=(VariableSyncModule&&) = delete;
 
 private:
+    static constexpr const char* LOGNAME = "ConstraintSolver";
```

Comment:
VariableSyncModule and Query have the same logname now

Reply:
yes, since the logName is used as a log space, I figured that we can have a single log space for all constraint solving related classes. That way it will be easier to filter